third_party/stb/stb_truetype.c

```diff
@@ -1,29 +1,20 @@
-/*-*- mode:c;indent-tabs-mode:nil;c-basic-offset:3;tab-width:8;coding:utf-8 -*-│
-│vi: set net ft=c ts=3 sts=3 sw=3 fenc=utf-8                                :vi│
+/*-*- mode:c;indent-tabs-mode:nil;c-basic-offset:2;tab-width:8;coding:utf-8 -*-│
+│vi: set net ft=c ts=2 sts=2 sw=2 fenc=utf-8                                :vi│
 ╞══════════════════════════════════════════════════════════════════════════════╡
+│ Copyright 2023 Justine Alexandra Roberts Tunney                              │
 │                                                                              │
-│  stb_truetype                                                                │
```

Comment:
Could you put the copyright of the original authors back? Even though this is a public domain library, it's a bad look for me to impose further restrictions on its use, even if they are as minor as the ISC license. My intent is for the STB code in this folder of the cosmopolitan repo to keep being public domain.

Reply:
Sure. I've reverted my changes.